packages/gasket-data/lib/index.d.ts

```diff
@@ -1,5 +1,7 @@
 declare module '@gasket/data' {
-    export interface GasketData {}
+    export interface GasketData {
+      config: { [key: string | number]: string | undefined };
```

Comment:
Although not the tightest type, this is already declared by the plugin that add the config prop [here](https://github.com/godaddy/gasket/blob/deec24a99256e6b875a08ec20ed83b40212a72e4/packages/gasket-plugin-config/lib/plugin.d.ts#L20-L22).

In your code, that expect the config plugin to have added this property, you can pull in the types by add towards the top of your file:
```
/// <reference types="@gasket/plugin-response-data"/>
```

Reply:
I think I might've taken the wrong point from your feedback here, but I updated the gasket plugin config type as well 😛